subsys/bluetooth/audio/csip_crypto.c

```diff
@@ -101,12 +99,12 @@ int bt_csip_sih(const uint8_t sirk[BT_CSIP_SET_SIRK_SIZE], uint32_t r,
 	 * result of sih.
 	 */
 
-	LOG_DBG("BE: res %s", bt_hex(res, sizeof(res)));
+	LOG_DBG("res %s", bt_hex(res, 16));
```

Comment:
This should be reverted to use `sizeof(res)`

Reply:
Actually it should be using the macro in this file (discussed above). I'll fix

Reply:
If your intention is to log the entire `res`, then you may as well use `sizeof` of the value you want to log :)